src/xpk/commands/cluster.py

```diff
@@ -146,6 +152,14 @@ def cluster_create(args) -> None:
   if install_kueue_on_cluster_code != 0:
     xpk_exit(install_kueue_on_cluster_code)
 
+  xpk_print('Verifying kjob installation')
+  verify_kjob_installed(args)
```

Comment:
Would it be better if these methods returned some results and you give the control of decision to exit app here?

Reply:
There is a similar discussion here: https://github.com/AI-Hypercomputer/xpk/pull/208#discussion_r1804347717

I would like to add here that if the method is named verify I would expect it to exit the app upon failing, if it would be something like check then I would expect it to return the result.

In this case I believe @pawloch00 should leave the code as is.